libember_slim/Source/glow.h

```diff
@@ -365,8 +365,8 @@ typedef enum EGlowFieldFlags
    GlowFieldFlag_TemplateReference  = 0x00080000,
    GlowFieldFlag_Connections        = 0x00000005,
 
-   GlowFieldFlag_All                = 0xFFFFFFFF,
-   GlowFieldFlag_Sparse             = -2
+   GlowFieldFlag_All                = (int)0xffffffff,
```

Comment:
Wouldn't the constants `-1` and `-2` have the desired effect regardless of word size?